_config.yml

```
@@ -107,7 +107,7 @@ asciidoc:
     prod-prev-ver: "7.39"
     prod-short: Che
     prod-upstream: Eclipse{nbsp}Che
-    prod-url: https://che-host:che-port
+    prod-url: https://che-host
```

And even with formatting?

```suggestion
    prod-url: https://_<che-host>_
```